pkgs/development/tools/coder/default.nix

```
@@ -100,7 +100,7 @@ stdenvNoCC.mkDerivation (finalAttrs: {
     homepage = "https://coder.com";
     license = lib.licenses.agpl3Only;
     mainProgram = "coder";
-    maintainers = with lib.maintainers; [ ghuntley urandom ];
+    maintainers = with lib.maintainers; [ ghuntley urandom kylecarbs ];
```

```suggestion
    maintainers = with lib.maintainers; [ ghuntley kylecarbs urandom ];
```
(alphabetical order)